packages/office-ui-fabric-react/src/demo/pages/ButtonPage/ButtonPage.tsx

```diff
@@ -55,9 +53,6 @@ export class ButtonPage extends React.Component<IComponentDemoPageProps, IButton
             <ExampleCard title='Primary Button' code={ ButtonPrimaryExampleCode }>
```

Comment:
We should update all files and text to refer to this as the "default" and not "normal", as the actual component name is DefaultButton.